examples/with-jest/package.json

```diff
@@ -18,10 +18,6 @@
     "@testing-library/react": "12.1.2",
```

Comment:
If we remove ESLint, let's also remove this line: https://github.com/vercel/next.js/blob/426afcfe6cae049f9a442e80e032884a9bcad585/examples/with-jest/package.json#L5